benchmark/auto/README.md

```diff
@@ -0,0 +1,48 @@
+# AUTO BENCHMARK
+
+Auto benchmark aims to generate testsets with synthetic Q&A pairs from raw passages, using as few human annotations as possible. To achieve this, we use LLM as a annotator and dataset quality inspector.
+
+## Usage
+
+1. Prepare your corpus in a JSON file, following the format
+
+```
+corpus.json: [
+      {"document": "Your document text here"}
+]
```

Comment:
In your sample corpus.json file, it seems that the format is list of string rather than what is said above in readme. 
Maybe it is: ["document paragraph1", "document paragraph2"...] ?

Reply:
Fixed.